lib/tic_tac_toe_api_web/controllers/page_controller.ex

```diff
@@ -1,22 +1,38 @@
 defmodule TicTacToeApiWeb.PageController do
   use TicTacToeApiWeb, :controller
 
+  @env Application.get_env(:tic_tac_toe_api, :env)
```

Comment:
It is strange to me that you are injecting an environment. Instead, why not inject a specific seed value?